.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "gems/base"]
+	path = gems/base
+	url = git@github.com:ActiveState/stackato-services-base.git
+	branch = 300049-password-leak-in-log-files
```

Comment:
@troytop I most have initialized the submodule with the branch name, I will update!